src/modules/core/components/Dialog/DialogSection.tsx

```diff
@@ -4,10 +4,11 @@ import { getMainClasses } from '~utils/css';
 
 import styles from './DialogSection.css';
 
-interface Appearance {
+export interface Appearance {
   theme?: 'heading' | 'sidePadding' | 'footer';
   align?: 'center' | 'right';
   border?: 'top' | 'bottom' | 'none';
+  margins?: 'small' | 'large';
```

Comment:
Please, remove the `margins` prop & create another core component `InfoSection` ( or something like this) where you reuse `DialogSection`. There you can just have the margin as you need it. I would recommend just have it without the prop even. Just give a default value for now (the one that you need).